thorlcr/graph/thgraphslave.cpp

```diff
@@ -1028,7 +1028,7 @@ class CThorCodeContextSlave : public CThorCodeContextBase, implements IEngineCon
     virtual bool allowDaliAccess() const
     {
         // NB. includes access to foreign Dalis.
-        return globals->getPropBool("Debug/@slaveDaliClient");
+        return globals->getPropBool("Debug/@slaveDaliClient") || job.getWorkUnitValueBool("slaveDaliClient", false);
```

Comment:
Except that would test globals "@slaveDaliClient" not "Debug/@slaveDaliClient", unless I am misreading the code